blaze_tensor/math/views/quatslice/Dense.h

```diff
@@ -1507,7 +1507,7 @@ template< typename AT       // Type of the dense quaternion
 BLAZE_ALWAYS_INLINE typename QuatSlice<AT,CRAs...>::SIMDType
    QuatSlice<AT,CRAs...>::load( size_t k, size_t i, size_t j ) const noexcept
 {
-   return quaternion_.load( quat(), k, i, j );
+   return quaternion_.load( j, i, k, quat() );
 }
```

Comment:
Is this really what you intended to do? `load` should receive the row index first, then everything else in decreasing dimensionality, i.e. I would expect seeing `load(I, quat(), k, j);`

Reply:
Ok, I take that back. I would expect what was there before... Why did you change this?